python/README.md

```diff
@@ -17,7 +17,7 @@
 
 # Iceberg Python
 
-pyiceberg is a python library for programmatic access to iceberg table metadata as well as to table data in iceberg format. It is a Python implementation of [iceberg table spec](https://iceberg.apache.org/spec/). Documentation is available at [https://pyiceberg.apache.org/](https://pyiceberg.apache.org/).
+pyiceberg is a python library for programmatic access to iceberg table metadata as well as to table data in iceberg format. It is a Python implementation of [iceberg table spec](https://iceberg.apache.org/spec/). Documentation is available at [https://py.iceberg.apache.org/](https://py.iceberg.apache.org/).
```

Comment:
I know that this was the working link. But when I click now, I get a 404 error. Can you please check once?

Reply:
Thanks for reporting. I also noticed this yesterday and pinged Apache INFRA: https://issues.apache.org/jira/browse/INFRA-23715

Reply:
Also created a new ticket: https://issues.apache.org/jira/browse/INFRA-23738

Reply:
It works again, thanks again @ajantha-bhat for pointing out 👍🏻